ansible/roles/postgres/tasks/main.yml

```diff
@@ -101,3 +108,15 @@
     - role::postgres
   notify:
     - Restart the postgres service
+
+- name: Import PostgreSQL identity map (pg_ident.conf)
```

Comment:
The docs say the following
> The pg_ident.conf file is read on start-up and when the main server process receives a SIGHUP signal. If you edit the file on an active system, you will need to signal the postmaster (using pg_ctl reload, calling the SQL function pg_reload_conf(), or using kill -HUP) to make it re-read the file.

Wouldn't this mean that we can just reload the service instead of restarting it here ?